static/js/modules/bar-charts.js

```diff
@@ -0,0 +1,269 @@
+'use strict';
+
+/* global module, d3 */
+var $ = require('jquery');
+var helpers = require('./helpers');
+var stripes =
+  '<svg width="10px" height="10px" viewBox="0 0 10 10" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">' +
+      '<defs>' +
+        '<pattern id="stripes" patternUnits="userSpaceOnUse" width="10" height="10">' +
+            '<path stroke="#979797" d="M-1.5,-1.5 L10.0108643,10.0108643" id="Line"></path>' +
+        '</pattern>' +
+      '</defs>' +
+  '</svg>';
+
+
+function GroupedBarChart(selector, data) {
+  this.element = d3.select(selector);
+  this.data = data;
+
+  this.margin = {top: 0, right: 20, bottom: 50, left: 40};
```

Comment:
Mmm I don't think so. They're the _future_ properties of the `svg` element created below, and are used to calculate various things before that's appended.